CMakeLists.txt

```diff
@@ -196,6 +196,11 @@ endif()
 option(BUILD_SHARED_LIBS "Build shared libraries" ${LIB_DEFAULT})
 option(EXPORT_ALL_SYMBOLS "Export all symbols form library" OFF)
 
+if (BUILD_SHARED_LIBS STREQUAL "OFF")
```

Comment:
I don´t like this, should be moved to some `toolchain` file you use for your specific case and not part of the main `CMakeLists.txt`

Reply:
Moved:

```diff
--- a/cmake/ConfigOptions.cmake
+++ b/cmake/ConfigOptions.cmake
@@ -199,6 +199,11 @@ if (BUILD_FUZZERS)
 
     set(BUILD_TESTING ON)
 
+    if (BUILD_SHARED_LIBS STREQUAL "OFF")
+        set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
+        set(CMAKE_CXX_FLAGS "-static ${CMAKE_CXX_FLAGS}")
+    endif()
+
     # A special target with fuzzer and sanitizer flags.
     add_library(fuzzer_config INTERFACE)
 
```